### DISCUSSION TITLE
A lot of mails in "queued messages"

### DISCUSSION THREAD

**Discussion by paulus80** [postalserver/postal#1603] [Help with using Postal]
Hello, i´m starting to use postal and i think that i´m doing someting incorrectly.

I have installed postal using the docker and i have started all correctly. I have created my first mailing campaigns using MailWizz. All seems works fine, but i looking in the "Messages - Outgoing Messages" i have this:

Status:sent - 17,635 messages
Status Held - 42 messages
Status Pending - 83,460 messages
Status softfail - 9,375 messages
Status hardfail - 23.200 messages
Status bounced - 0 messages

Al in the top of the page i have this 

- 93688 queued messages

I don´t know how interpretate it. What includes exactly the "queued messages" area? I suppose that should be this:

queued messages = Pending + Softfail 

Is this correct?

My other question is, why i have all this messages as pending? What must i do for process all this messages?

I have downloaded the database and i have 83.405 messages witho "0" attemps. It seems that Postal hasn´t tryed to send all this messages.

Thanks for your help.

**Comment by willpower232** [comment]
What is the output of `postal status`?

**Comment by paulus80** [comment]
![image](https://user-images.githubusercontent.com/47594908/134645917-c6cfb204-523c-4201-9187-cd0eba5f5077.png)

**Reply by willpower232** [reply]
It is likely that you need to scale up the amount of workers as you have a lot of messages to deal with, have a look at https://github.com/postalserver/postal/discussions/1461#discussioncomment-1119403

**Comment by paulus80** [comment]
Hi, i scalled it when i louched the campaigns, and doing it i obtained a better sender speed.

I´ll go to test one thing now. I´ll drop the table "queued_messages", and i´ll send the same campaing increasing the workers since the first moment. I´ll post here the results when finish it.

**Comment by paulus80** [comment]
Hello, i have sended a new campaign increasing the workers since the first moment, i have the same problem. I have 68467 queued messages. I have downloaded the table "queued_messages" for see how many attemps of send the messages has tryed the system, and i have this:

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/PABLOG~1/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/PABLOG~1/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



Attempts | Total
-- | --
0 | 60823
1 | 2210
2 | 1616
3 | 1286
4 | 1223
5 | 860
6 | 438
7 | 11
Total general | 68467



</body>

</html>

What do you think that i am doing incorrectly?

Thanks

**Reply by willpower232** [reply]
You would have to look at the messages to see why they failed and needed retrying.
